### DISCUSSION TITLE
Document build issues

### DISCUSSION THREAD

**Discussion by hongshui3000** [zephyrproject-rtos/zephyr#34129] [Q&A] [answered]
I found that it seems that Zephyr's document build does not completely force the use of utf-8 to process text. At least I found no on windows. What troubles me is that I can’t use other languages such as (Chinese) to add my own documents

**Comment by nashif** [comment]
@gmarull  Can you please take a look.

**Comment by gmarull** [comment, marked as answer]
I'll take a look. Related issue: https://github.com/zephyrproject-rtos/zephyr/issues/34076
